app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine is outdated.")
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError("Visitor is not wearing a mask.")
```

Comment:
if I know, when we raises errors, we need output name errors
`raise OutdatedVaccineError("OutdatedVaccineError")`
![image](https://github.com/user-attachments/assets/14f1616a-6ff7-4c87-9ca7-9e4dedbf109c)

Reply:
I'm not sure about that. In case of an error, I get the name of the error and its description in the terminal.

Reply:
This is good, we need text of error, to explain user why did this error raised(His solution is good)